docs/sources/reference/cli/environment-variables.md

```diff
@@ -14,13 +14,28 @@ The following environment variables are supported:
 
 * `GODEBUG`
 * `HTTP_PROXY`
+* `HTTPS_PROXY`
+* `NO_PROXY`
 * `PPROF_MUTEX_PROFILING_PERCENT`
 * `PPROF_BLOCK_PROFILING_RATE`
 * `GOMEMLIMIT`
-
+* `AUTOMEMLIMIT`
+* `GOGC`
+* `GOMAXPROCS`
+* `GOTRACEBACK`
 
 Refer to the [Go runtime][runtime] documentation for more information about Go runtime environment variables.
 
+# Windows Service Configuration
+
+To configure any of the available environment variables when running {{< param "PRODUCT_NAME" >}} as a Windows Service, open `regedit`, the Windows Registry Editor, and navigate to the registry key `HKEY_LOCAL_MACHINE\SOFTWARE\GrafanaLabs\Alloy`. Under this key will be an `Environment` multi-string value where any desired environment variables can be provided on separate lines.
+
+For example, if you wanted to configure `NO_PROXY` and `GOGC` you would enter the following as the `Environment` value. Refer to our [Windows] configuration documentation for more details.
```

Comment:
I am not sure this example belongs here. I'd suggest it's in the OS-specific config topic - ie in the Windows topic.